api/files/serializers.py

```diff
@@ -380,3 +385,24 @@ def absolute_url(self, obj):
 
     def get_absolute_url(self, obj):
         return self.self_url(obj)
+
+    def get_download_link(self, obj):
+        return get_file_download_link(
+            self.context['view'].get_file(), version=obj.identifier,
```

Comment:
I'm a little wary of how this might affect performance. Won't this result in an additional query for each version in the response? @laurenrevere Would you mind seeing how response times and # queries are affected by this in BAPI locally?